### DISCUSSION TITLE
Problems in Building Block and Simulations

### DISCUSSION THREAD

**Discussion by AParisotto** [Open-Systems-Pharmacology/Forum#884] [Questions & Problems] [answered]
Hello everyone
In Pk-SIM it is possible to visualize the building block.
But the block where the simulations are located is no longer available, and it is not possible to access the simulations already made.
I have already used the shortcut keys Ctrl + Shif + S and the "simulations block" did not appear
What can I do to correct this problem?

**Comment by msevestre** [comment, marked as answer]
@AParisotto 
try this

https://github.com/Open-Systems-Pharmacology/Forum/issues/423#issuecomment-552640658
